Makefile

```diff
@@ -0,0 +1,5 @@
+docker/build:
+	docker build -t hercules:latest .
+
+docker/run_attached:
+	docker run --rm -p 8080:8080 hercules:latest
```

Comment:
Should we set the port `8080` as the `PORT` environment variable to be used by the app in the Makefile or leave it in the Dockerfile?

Reply:
The Makefile is mainly for development purposes. And considering that we would just be running the bundled Dockerfile and Makefile in development, it makes sense to just hardcode the ports here.

Reply:
Sounds good!

Also, how do we connect the app running in the container to the local DB?
While testing the current Dockerfile for development, I couldn't get the DB connected.
Should the DB be run in a separate container and both of these containers interact to establish the connection?

P.S. I have read just about mounting volumes yet; haven't studied container-container interaction.